### PR TITLE
Update the README to explain how to deploy.

### PR DESCRIPTION
**Pull request by kurtwheeler**
This PR also changes the AWS load balancer in the nginx configuration.

### Motivation

I have created infrastructure within the Greene Lab AWS account to run the Cognoma backend. This included creating a new internal load balancer which needed to be specified in the Nginx config. I also wanted to explain how to deploy changes to this service.

### API changes

None.

### Implementation Notes

This should be pretty straightforward. Let me know if there's any confusion.

### Functional Tests

I ran this command from my terminal with this output:
```
curl https://api.cognoma.org/classifiers; echo
{"count":0,"next":null,"previous":null,"results":[]}
```

### Screenshots/Logs

N/A

### CODE REVIEW

**Comment by dhimmel** [comment]
@kurtwheeler it took me a while to understand what your semicolon was doing... but now I see echo is just a separate command to print a newline. I used the following (not that it's better, just FYI):

```sh
curl --silent https://api.cognoma.org/classifiers | python -m json.tool
```

File: README.md [hunk 1]
```diff
@@ -42,10 +42,70 @@ To load data, again with service up run:
...
+ECS provides a private container registry called the
+[Ec2 Container Repository (ECR)](https://aws.amazon.com/ecr/).
+To be able to push Docker images to this repository you will first need to
+get a login with:
+```sh
+aws ecr get-login --region us-east-1
+```
+and then run the output of that command. It will look something like:
+```sh
+docker login -u AWS -p <A_GIANT_HASH> -e none https://934349674827.dkr.ecr.us-east-1.amazonaws.com
```
Comment (dhimmel) [inline README.md]: can we use long arguments when available in docs for clarity. i.e. `--password`, `--username`, `--email`

Comment (kurtwheeler) [inline README.md]: I'll make that change for the other commands in this README, however this specific command is actually outputted by the AWS CLI tool and the only reason I have it here is for reference. I'd like it to match the output of that command as much as possible, so I think it makes sense to leave it like this.

File: README.md [hunk 2]
```diff
@@ -94,7 +94,7 @@ docker push 934349674827.dkr.ecr.us-east-1.amazonaws.com/cognoma-core-service:la
 
 Run these commands:
 ```
-docker build -t cognoma-nginx -f config/prod/Dockerfile_nginx .
+docker build -tag cognoma-nginx -file config/prod/Dockerfile_nginx .
```
Comment (dhimmel) [inline README.md]: Double hyphen for long args `--tag`?

Comment (dhimmel) [inline README.md]: It's only very rare and troll programs that use single hyphen for long args.

Comment (kurtwheeler) [inline README.md]: Yes of course! Whoops.

I definitely agree that programs that use single hyphens for long args are trolls though haha

**Comment by kurtwheeler** [comment]
@dhimmel I've squashed my commits into a single commit message. I can't click merge, so I guess you've gotta do that for me.

**Comment by dhimmel** [comment]
> @dhimmel I've squashed my commits into a single commit message. I can't click merge, so I guess you've gotta do that for me.

BTW GitHub can do this automatically... so no need to rebase yourself unless you want to preserve multiple commits.

**Comment by kurtwheeler** [comment]
I noticed that today while reviewing a different PR. That's good to know cause git could have made that a lot easier on the command line.

**Pull request merged**